### PR TITLE
asimov: replace `plist` with `service`

### PR DESCRIPTION
**Pull request by edith007**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

This PR aims to solve the issue https://github.com/Homebrew/homebrew-core/issues/79367

### CODE REVIEW

File: Formula/asimov.rb [hunk 1]
```diff
@@ -14,25 +14,9 @@ def install
     bin.install buildpath/"asimov"
   end
 
-  plist_options startup: true, manual: "asimov"
```
Comment (carlocab) [inline Formula/asimov.rb:17]:
```suggestion
```
Could you squash this into your existing commit please? Thanks!

**Comment by SMillerDev** [comment]
There is no interval support in service blocks yet so this won't work.

**Comment by edith007** [comment]
> There is no interval support in service blocks yet so this won't work.

Oh thanks for letting me know. As i am new to `Homebrew` i forget to check the prerequisite. Can you please help me with this? 🙇🏻

**Comment by SMillerDev** [comment]
The code you removed has a StartInterval key, but the service block does not. You didn't add it, and you couldn't if you wanted to because service blocks don't support an interval yet.

**Comment by edith007** [comment]
So should i make the changes in the files which doesn't have StartInterval key??

**Comment by SMillerDev** [comment]
You should not be changing any plist blocks where you can't translate all keys to service methods.

**Comment by edith007** [comment]
Hi @SMillerDev! 👋🏻 
Now that `asimov` is has interval support. Can we close this PR?? 

I have been searching for the formulae to use the OS agnostic `service do` block, can we make changes in `cassandra@2.1` & `cassandra@2.2`?? 🤔 🙇🏻

**Pull request closed without merging**